src/gui/synthesispopover.cpp

```diff
@@ -1290,14 +1290,21 @@ void SynthesisPopover::onOpenMiscellaneousMenu(bool checked) {
     connect(exitAction, &QWidgetAction::triggered, this, &SynthesisPopover::onExit);
     menu->addAction(exitAction);
 
-    if (_debugMode) {
+    if (_debugCrash) {
         // Test crash
         QWidgetAction *crashAction = new QWidgetAction(this);
         MenuItemWidget *crashMenuItemWidget = new MenuItemWidget("Test crash");
         crashAction->setDefaultWidget(crashMenuItemWidget);
         connect(crashAction, &QWidgetAction::triggered, this, &SynthesisPopover::onCrash);
         menu->addAction(crashAction);
 
+        // Test crash server
+        QWidgetAction *crashServerAction = new QWidgetAction(this);
+        MenuItemWidget *crashServerMenuItemWidget = new MenuItemWidget("Test crash server");
```

Comment:
```suggestion
        MenuItemWidget *crashServerMenuItemWidget = new MenuItemWidget("Emulate server crash");
```

Or "Crash the server".

Reply:
ok